packages/cli/src/util/dev/server.ts

```diff
@@ -2156,7 +2156,7 @@ export default class DevServer {
       .catch(() => false);
 
     if (isNpxAvailable) {
-      command = `npx --no-install ${command}`;
+      command = `npx --no ${command}`;
```

Comment:
Seems like we need to support both flags so it works with old and new versions of npx?

Reply:
Yeah I forgot that 😅
`--no` is available since npm 7, which is bundled since Node 15. That means this change can only be made when Node 14 retires, which is 2023-04-30...

I will close this, but perhaps leave a note or so.

Reply:
Do we know if `--no-install` is just deprecated or is it completely removed in newer versions of npm?

Reply:
It's just deprecated. I don't think npm is going to remove it any time soon.